pdns/recursordist/taskqueue.hh

```diff
@@ -43,15 +50,23 @@ struct ResolveTask
 {
   DNSName d_qname;
   uint16_t d_qtype;
+  // Deadline is not part of index and <
```

Comment:
`and <`??

Reply:
`operator<`(), used when comparing `ResolveTask`s